docs/.docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+Contributing to Mayflower Documentation
+=========================
+
+Documentation for the overall Mayflower design system, including guiding principles hosted by Gitbook. 
+
+
+From Gitbook Editor:
+------------------------------------
+
+1. [Create](https://www.gitbook.com/join) a Gitbook account and ask for edit permission.
+2. Go to https://massds.gitbook.io/mayflower/.
+3. Click on `master` version dropdown and switch to the `docs/integration` version.
+4. Click on the `Edit` button to start making changes.
+5. Click on `docs/integration` and create a new "version" from the `docs/integration` version for your changes and name it `docs/[my_version]`
+This creates a `docs-[my_version]` branch in the Github repo.
+6. Click on the "save" button save as a draft. This is similar to stashing your changes in git, you can rechieve a draft from `Archived Drafts`.
+7. Click on the "publish" button to push the changes to the github branch. Please make sure you are on your feature version and ***never*** "publish" from the `master` version.
+8. Open a [Pull Request](/compare) and set the base to `docs/integration` and have your code reviewed.
```

Comment:
where is the pull request link supposed to send me? that one `/compare` leads to a 404